<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandlePropertyWidget</class>
 <widget class="QWidget" name="HandlePropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="show3DWidget">
     <property name="text">
      <string>Show Point</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="pointLabel">
     <property name="text">
      <string>Point</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="pqDoubleLineEdit" name="worldPositionX"/>
   </item>
   <item row="1" column="2">
    <widget class="pqDoubleLineEdit" name="worldPositionY"/>
   </item>
   <item row="1" column="3">
    <widget class="pqDoubleLineEdit" name="worldPositionZ"/>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="pickLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Note: Use 'P' to pick 'Point' on mesh or 'Ctrl+P' to snap to the closest mesh point</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QPushButton" name="useCenterBounds">
     <property name="text">
      <string>Center on Bounds</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqDoubleLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pqDoubleLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
